src/components/analytics-monsido/index.tsx

```diff
@@ -4,7 +4,9 @@ const AnalyticsMonsido: FC = () => {
   const { hostname } = location;
   const isFellesDatakatalog = [
     'data.norge.no',
-    'fellesdatakatalog.digdir.no'
+    'fellesdatakatalog.digdir.no',
+    'staging.fellesdatakatalog.digdir.no',
+    'demo.fellesdatakatalog.digdir.no'
```

Comment:
Undersøk hva som blir konsekvensen av at monsido script blir lastet for disse miljøene og hvordan det påvirker statistikkene i monsido. Vi ønsker ikke at staging eller demo skal påvirke prod tallene.

Reply:
Godt poeng! Tror det er best å separere disse statistikke. Skal undersøke

Reply:
Avgjørelsen ble sånn at vi tar utgangspunkt i prod foreløpig. Har fjernet subdomenene.